kenya_compliance/kenya_compliance/overrides/server/stock_ledger_entry.py

```diff
@@ -258,7 +271,14 @@ def get_stock_entry_movement_items_details(
 ) -> list[dict]:
     items_list = []
 
-    for item in records:
+    for i in records:
```

Comment:
This exception handling isn't needed since the properties being accessed of the item will always be bound.